lib/get-hook.js

```diff
@@ -2,14 +2,23 @@
 
 var _ = require('lodash');
 var execSync = require('shelljs').exec;
+var path = require('path');
 var stripBom = require('strip-bom');
 var map = require('map-stream');
 var hookArgs = process.env.HOOK_ARGS ? process.env.HOOK_ARGS.split('\u263a') : [];
 
 function getIndexed() {
-  return _.compact(execSync('git diff --cached --name-only --diff-filter=ACM', {
+  var gitParentDir = process.env.GITDIR || process.cwd();
```

Comment:
Let's avoid using env vars to control this. See [this comment](https://github.com/therealklanni/guppy-cli/pull/11#issuecomment-219094619).

In summary, I'd rather see this done using a package that can automatically determine the project/app root.